cibuildwheel/oci_container.py

```diff
@@ -71,6 +71,18 @@ def options_summary(self) -> str | dict[str, str]:
                 "disable_host_mount": str(self.disable_host_mount),
             }
 
+    def __eq__(self, value: object) -> bool:
```

Comment:
Why is this being defined? It looks like what data classes generate for you to me. It should at least have a commend explaining why this is required.

Reply:
hm. good point. I defined `__hash__` because it was needed for the build steps computation, so instinctively defined this too. Actually, if I just define the create_args as a tuple, I can get rid of both and let dataclass do it for me.